slider/replay.py

```diff
@@ -576,6 +636,62 @@ def parse(cls, data, *, library=None, client=None, save=False):
             **mod_kwargs,
         )
 
+    @lazyval
+    def hits(self):
+        b = self.beatmap
+        rdata = self.actions
```

Comment:
it is confusing to give this a new name when binding locally, could we just call this `actions`, or rename both here and on the instance to `replay_data`?